components/41-organisms/filter/_filter.scss

```diff
@@ -29,7 +29,7 @@
       }
 
       > * + * {
-        margin-left: gutter() / 2;
+        margin-left: gutter() *.5;
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.